### PR TITLE
Fogl 277 coap server updates statistics READINGS and DISCARDED every 5 seconds

### PR DESCRIPTION
**Pull request by ashwinscale**

### CODE REVIEW

File: src/python/foglamp/device/coap/sensor_values.py [hunk 1]
```diff
@@ -39,16 +41,30 @@ class SensorValues(aiocoap.resource.Resource):
     """CoAP handler for coap://readings URI"""
 
     _CONNECTION_STRING = "dbname='foglamp'"
+
     # 'postgresql://foglamp:foglamp@localhost:5432/foglamp'
 
+    _num_readings = 0
+    _num_discarded_readings = 0
```
Comment (Terris-Dianomic) [inline src/python/foglamp/device/coap/sensor_values.py]: Please move these to init, initialize them as self._num_readings, and give them a short docstring.

File: src/python/foglamp/device/coap/sensor_values.py [hunk 2]
```diff
@@ -70,12 +86,15 @@ def register_handlers(self, resource_root):
...
 
+        self._num_readings = self._num_readings + 1
+
         # Required keys in the payload
         try:
             payload = loads(request.payload)
             asset = payload['asset']
             timestamp = payload['timestamp']
         except:
+            self._num_discarded_readings = self._num_discarded_readings + 1
```
Comment (Terris-Dianomic) [inline src/python/foglamp/device/coap/sensor_values.py]: Use += 1

Comment (ashwinscale) [inline src/python/foglamp/device/coap/sensor_values.py]: done

File: src/python/foglamp/device/coap/sensor_values.py (hunk 1 above)
Comment (ashwinscale) [inline src/python/foglamp/device/coap/sensor_values.py]: done

**Pull request merged**